timewarp_gl/plugin.cpp

```diff
@@ -580,8 +599,21 @@ class timewarp_gl : public threadloop {
 			{std::chrono::high_resolution_clock::now()},
 			{std::chrono::nanoseconds(elapsed_time)},
 		}});
+
+#ifndef NDEBUG
+		if (log_count > LOG_PERIOD) {
+			log_count = 0;
+		} else {
+			log_count++;
+		}
+#endif
 	}
 
+#ifndef NDEBUG
+	size_t log_count = 0;
+	size_t LOG_PERIOD = 20;
```

Comment:
I think it would be clearer to change LOG_PERIOD to LOG_START_ITER or something of the sort.

Reply:
Maybe there is some confusion. It doesn't just start after 20, it records every 20. So 20, 40, 60, ... get recorded.

Reply:
Oh yeah count is reset on 605 every time it passes log_period